### ISSUE TITLE
Confusing naming

### ISSUE THREAD

**Issue by veqtor** [max-liu-112/SRGANs-Spectral-Regularization-GANs-#1]
While the paper is about spectral regularization, all of the code maintains the SN / Spectral Normalization naming, it would be nice, just for clarity if it was changed to SR / Spectral Regularization.

**Comment by mgirardot** [comment]
It seems that the modifications are not pushed to the repo yet. I don't see any operation related to the equations in the paper. Maybe after ICCV2019 ?
Very interesting paper.

**Comment by max-liu-112** [comment]
hi, Michael,
Thank you for you contacting.
The following link
https://github.com/max-liu-112/SRGANs is our proposed SRGANs, training on
ImageNet. Actually, few modification need to be made based on SNGANs.
Specifically, the ‘updated.py’ involves the sr method, using dynamic
compensation.
Dynamic compensation is introduced in the latest version
https://arxiv.org/pdf/1908.10999.pdf.


Michael Girardot <notifications@github.com>于2019年10月31日 周四下午8:16写道：

**Comment by mgirardot** [comment]
Great, thank you for the clarification. I could not find the SR specific code. Could you maybe add a pointer in the readme ?

**Comment by max-liu-112** [comment]
ok， I will do it later. Now I am out for dinner.

Michael Girardot <notifications@github.com>于2019年10月31日 周四下午10:04写道：

**Comment by max-liu-112** [comment]
We update the read.md. We hope it works fine for you. If any more
questions, please contact me. Thanks

Michael Girardot <notifications@github.com>于2019年10月31日 周四下午10:04写道：

**Comment by mgirardot** [comment]
Thank you. I am following the equations in the paper and the code to try to adapt it to my code base (Keras/tensorflow).
